var/spack/repos/builtin/packages/metis/package.py

```diff
@@ -172,25 +191,16 @@ def install(self, spec, prefix):
         if '+gdb' in spec:
             options.append('-DGDB:BOOL=ON')
 
-        metis_header = join_path(source_directory, 'include', 'metis.h')
-        if '+idx64' in spec:
-            filter_file('IDXTYPEWIDTH 32', 'IDXTYPEWIDTH 64', metis_header)
-        if '+real64' in spec:
-            filter_file('REALTYPEWIDTH 32', 'REALTYPEWIDTH 64', metis_header)
-
-        # Make clang 7.3 happy.
-        # Prevents "ld: section __DATA/__thread_bss extends beyond end of file"
-        # See upstream LLVM issue https://llvm.org/bugs/show_bug.cgi?id=27059
-        # and https://github.com/Homebrew/homebrew-science/blob/master/metis.rb
-        if spec.satisfies('%clang@7.3.0'):
-            filter_file('#define MAX_JBUFS 128', '#define MAX_JBUFS 24',
-                        join_path(source_directory, 'GKlib', 'error.c'))
-
         with working_dir(build_directory, create=True):
             cmake(source_directory, *options)
             make()
             make('install')
 
+            # FIXME: On some systems, the installed binaries for METIS cannot
+            # be executed without first being read.
+            ls = which('ls')
+            ls('-all', prefix.bin)
```

Comment:
i have not tested this, but it looks like `-all` is not there on Darwin: https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/ls.1.html